securedrop_export/export.py

```diff
@@ -119,6 +124,41 @@ def extract_tarball(self):
             self.exit_gracefully(msg, e=e)
 
 
+    def check_usb_connected(self):
+        # Rely on the output of lsusb on the bus assigned to. We might need to make this variable configurable
+        # In the future and extracted from config.json
+        p = subprocess.check_output(["lsusb", "-s", PCI_BUS_ID])
+        # Empty string means a likely wrong PCI_BUS_ID
+        if p == "":
+            msg = "ERROR_USB_CHECK"
+            self.exit_gracefully(msg)
+        n_usb = len(p.rstrip().split("\n"))
```

Comment:
the usb check fails for me unless i sub in `p.decode("utf-8")` for `p` here.

Reply:
what version of python are you using? Getting the following error with 3.5.7:
 ```
>       n_usb = len(p.decode("utf-8").rstrip().split("\n"))
E       AttributeError: 'str' object has no attribute 'decode'
```

Reply:
Nevermind my commend above, this was due to the test strings not being the correct format :man_facepalming: , this is now fixed.